projects/demo/src/modules/experimental/pin/examples/3/index.html

```diff
@@ -0,0 +1,39 @@
+<button
+    tuiPin
+    [open]="a"
+    (click)="a = !a"
+>
+    <img
+        alt="avatar"
+        src="assets/images/avatar.jpg"
+    />
+</button>
+
+<button
+    tuiPin
+    [open]="b"
+    [style.left.rem]="5"
+    (click)="b = !b"
+>
+    <tui-icon icon="assets/images/atm.svg"></tui-icon>
+</button>
+
+<button
+    tuiPin
+    class="link"
+    [open]="c"
+    [style.left.rem]="10"
```

Comment:
Why not `display: flex` + `gap` ?

Reply:
Because selected pins are wider so it would jump around on clicks.